explanations/dispatchers.ipynb

```diff
@@ -0,0 +1,688 @@
+{
```

Comment:
<p>Add a sentence in between the two paragraphs talking about memory usage and computational efficiency?</p><p> </p><ul><li>for dense variables, need only to store a row vector for each variable, then take the product: Less storage now, more computations, more storage for results </li><li>for continuous variables, need to store one row for each valid state x choice combination: More storage now (6 elements instead of 4), less computations , lower storage space for results (both 3 instead of 4)</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenSourceEconomics/lcm/pull/68/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>